twilio-keycloak-provider/.github/workflows/build.yml

```
@@ -0,0 +1,29 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+      - 2-initial-keycloak-sms-authenticator
+  pull_request:
+    branches:
+      - main
```

The fact that you found it beneficial to specifically mention this branch during development suggests that we might be overly restrictive here. Maybe we should just run the build and tests on all branches?

```suggestion
on:
  push:
  pull_request:
```

---

Good call.